modules/mobile/spec/workers/vet360_linking_spec.rb

```diff
@@ -7,13 +7,23 @@
   let(:user) { create(:user, :loa3) }
 
   context 'when linking request is successfully made' do
-    it 'logs the user id and transaction id that linked an account with vet360' do
+    it 'logs the user id, transaction id, and creates a mobile user that linked an account with vet360' do
       VCR.use_cassette('mobile/profile/init_vet360_id_success') do
         expect(Rails.logger).to receive(:info).with(
           'Mobile Vet360 account linking request succeeded for user with uuid',
           { user_uuid: user.uuid, transaction_id: 'd8951c96-5b8c-42ea-9fbe-e656941b7236' }
         )
         subject.perform(user.uuid)
+        expect(Mobile::User.where(icn: user.icn)).to exist
+      end
+    end
+
+    it 'increments mobile user vet360 linked attempts' do
```

Comment:
I think it could be worth adding a check for marking the user complete. Could be part of this test or separate. As part of the test, you can manually set the vet360 id, then confirm that `subject.perform` changes `vet360_id` from false to true.

Reply:
Ok added related tests to the relevant request specs